controllers/kafka.go

```diff
@@ -0,0 +1,40 @@
+package controllers
+
+import (
+	"encoding/json"
+	"net/http"
+	"zestream-server/configs"
+	"zestream-server/utils"
+
+	"github.com/gin-gonic/gin"
+)
+
+type Video struct {
+	ID   string `json:"id"`
+	Src  string `json:"src"`
+	Type string `json:"type"`
+}
+
+type Body struct {
+	Video Video `json:"video"`
+}
```

Comment:
We should create a new package `types/types.go` that will hold all the structs like these and the Response types as well